gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +177,29 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        plateau_models = {}
+        coverage_model = {}
+        for region in GENOMIC_REGIONS:
+            logger.info("Building plateau and coverage models...")
+
+            # Check if the training dataset exist.
+            check_resource_existence(
+                input_pipeline_step="--create-training-set",
+                input_resources=training_resources.values(),
+                overwrite=overwrite,
+            )
+
+            # Build plateau and coverage models
+            coverage_model[region], plateau_models[region] = build_models(
```

Comment:
I will add a note at the next part that only coverage model built for autosomes/pseudoautosomal regions will be used.

Reply:
also, if we're planning to only use the coverage model from the autosomes/non-par regions, we should unnest this code from within this loop